frontend/src/metabase-lib/lib/metadata/Field.ts

```diff
@@ -420,6 +426,18 @@ class FieldInner extends Base {
     });
   }
 
+  clone(metadata) {
+    if (metadata instanceof Field) {
+      throw new Error("`metadata` arg must be a plain object");
+    }
```

Comment:
Should we maybe use another name instead of `metadata` here? When seeing `metadata` in `metabase-lib` I always expect it to be the `Metadata` class instance, so this `metadata instanceof Field` check was surprising

Reply:
Yep, good call. Updated.